routes/shopifyAuth.js

```diff
@@ -104,6 +112,436 @@ module.exports = function createShopifyAuthRoutes({
         console.error('🔴 Error storing shop access token', error);
         next(error);
       }
+    },
+
+    enableCookies(request, response) {
+      const HEADING = 'Enable cookies';
+      const BODY = 'You must manually enable cookies in this browser in order to use this app within Shopify.'
+      const FOOTER = `Cookies let the app authenticate you by temporarily storing your preferences and personal
+information. They expire after 30 days.`
+      const ACTION = 'Enable cookies';
+
+      const { query, baseUrl } = request;
+      const { shop } = query;
+
+      if (shop == null) {
+        return response.status(400).send('Expected a shop query parameter');
+      }
+
+      response.send(`<!DOCTYPE html>
```

Comment:
Is there a better way/place to do/put this HTML?

This same HTML (with different interpolation syntax) is in this repo, shopify_app and koa-shopify-auth. The same situation already exists with the HTML redirect code, so it's not a new problem, but wasn't sure if anyone had better ideas...